{
	"candidate": "CVE-2020-13332",
	"description": "** REJECT ** DO NOT USE THIS CANDIDATE NUMBER. ConsultIDs: none. Reason: This candidate was withdrawn by its CNA. Further investigation showed that it was not a security issue. Notes: none.",
	"priority": "untriaged",
	"references": [
		"https://www.cve.org/CVERecord?id=CVE-2020-13332"
	],
	"public_date": "2020-10-07 14:15:00 UTC",
	"packages": {
		"bionic": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"devel": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"esm-apps/xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected"
			}
		},
		"focal": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"groovy": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"precise/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty/esm": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"upstream": {
			"gitlab": {
				"name": "gitlab",
				"status": "needs-triage"
			}
		},
		"xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "not-affected"
			}
		}
	}
}
